financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/Color.kt

```diff
@@ -26,6 +26,8 @@ internal val Neutral900 = Color(0xff21252C)
 internal val Brand50 = Color(0xffF7F5FD)
 internal val Brand400 = Color(0xff8D7FFA)
 
+internal val Attention200 = Color(0xffFCAF4F)
```

Comment:
Should we align with PaymentSheet here, which also shows a `Test Mode` badge but in a different color? It uses `#FFDE94`, although I’m not sure if that matches to a Sail color.